traits/api.py

```diff
@@ -230,3 +229,8 @@
         AbstractViewElement.register(ViewElement)
 except ImportError:
     pass
+
+# Backward compatibility for comparison mode constants.
```

Comment:
Is this just for backward compatibility? E.g., should we recommend that people use `ComparisonMode.no_compare` in preference to `NO_COMPARE` going forward? If so, we should update the documentation accordingly.

Reply:
Yes, I think it makes sense to encourage the use of the Enums.

Reply:
Okay. I added a note to that effect to #695.